src/django/api/serializers/v1/opensearch_common_validators/countries_validator.py

```diff
@@ -0,0 +1,25 @@
+from api.serializers.v1.opensearch_validation_interface \
+    import OpenSearchValidationInterface
+import re
+
+
+class CountryValidator(OpenSearchValidationInterface):
+    def validate_opensearch_params(self, data):
+        errors = []
+        countries = data.get('country')
+
+        if not countries:
+            return errors
+
+        if isinstance(countries, str):
+            countries = [countries]
+
+        for country in countries:
+            if not re.match(r'^[A-Z]{2}$', country):
```

Comment:
I suppose it would be better to create a constant for the regex pattern.

Reply:
Done.